### PR TITLE
Add a default robots.txt

### PR DESCRIPTION
**Pull request by carakas**

### CODE REVIEW

File: robots.txt [hunk 1]
```diff
@@ -0,0 +1,5 @@
+User-agent: *
+Disallow: /private
+Disallow: /api
+Disallow: /backend/ajax
+Disallow: /backend/cronjob
```
Comment (WouterSioen) [inline robots.txt]: I'd also add /frontend/ajax, /api and /install

Comment (carakas) [inline robots.txt]: fixed in 31623200c8bfafb3f3c3fb22651919b7d3252603

**Comment by WouterSioen** [comment]
I don't know a lot about SEO, so I'll wait to merge this for now (until further feedback)

**Comment by jeroendesloovere** [comment]
Don't know if we should add ```private``` in the list.

Because
1. Google has no knowledge of "/private", so the chance that it will index it is probably zero.
2. This is encouraging hackers, because they now know where the login is.

**Comment by WouterSioen** [comment]
the private pages also have a `<meta name="robots" content="noindex, nofollow" />`  meta tag, so this should avoid indexing by default.

**Comment by carakas** [comment]
@jeroendesloovere fixed in 180c19c

**Comment by WouterSioen** [comment]
if there is no additional feedback, I'll merge this one.

**Pull request merged**